src/eml2_1/PropertyKind.h

```diff
@@ -81,6 +81,11 @@ namespace EML2_1_NS
 		 */
 		DLL_IMPORT_OR_EXPORT bool isParentPartial() const;
 		
+		/**
+		* Please do not use: not implemented yet.
+		*/
+		DLL_IMPORT_OR_EXPORT virtual std::string getUomAsString() const final;
```

Comment:
I don't think it should be virtual since it is final.

Reply:
Done.